tsconfig.json

```diff
@@ -1,14 +1,19 @@
 {
   "compilerOptions": {
-    "allowJs": true,
     "esModuleInterop": true,
     "moduleResolution": "node",
-    "outDir": "lib",
-    "sourceMap": true,
+    "declaration": true,
+    "declarationDir": "types",
     "strict": true,
-    "target": "es2020"
+    "target": "es2020",
+    "skipLibCheck": true,
```

Comment:
we should only enable this if there's an issue with a dependency we don't control

Reply:
not sure if it is the source of the problem but NodeJS throws an error like:

Cannot find namespace 'NodeJS'